### PR TITLE
Release 4.10.0

### PR DESCRIPTION
**Pull request by johnhwhite**
- Added validator option for cells. [#134](https://github.com/blackbaud/skyux-ag-grid/pull/134)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 4.10.0
```
Comment (blackbaud-johnly) [inline CHANGELOG.md:1]:
```suggestion
# 4.10.0  (2021-09-08)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
...
+
+- Added validator option for cells. [#134](https://github.com/blackbaud/skyux-ag-grid/pull/134)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md:3]:
```suggestion
- Added a validator option for cells. [#134](https://github.com/blackbaud/skyux-ag-grid/pull/134)
```

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-release-4.10.0](https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-release-4.10.0)

**Review by blackbaud-johnly — APPROVED** [review]

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-release-4.10.0](https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-release-4.10.0)

**Pull request merged**